### PR TITLE
throttle_utils:support blockdev_reopen function

### PR DESCRIPTION
**Pull request by qingwangrh**
ID:2032284
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
python3 ConfigTest.py --guestname=RHEL.8.6.0 --testcase=throttle_operation_test.group_move.to_normal --machines=q35 --clone=no

(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.throttle_operation_test.group_move.to_normal.q35: PASS (196.85 s)

**Comment by qingwangrh** [comment]
@YongxueHong @ybduan Please help to review. thanks.

File: provider/throttle_utils.py [hunk 1]
```diff
@@ -141,7 +141,10 @@ def change_throttle_group(self, image, group_id):
         file = throttle_blockdev.get_param("file")
         args = {"driver": "throttle", "node-name": node_name, "file": file,
                 "throttle-group": group_id}
-        self._monitor.x_blockdev_reopen(args)
+        if self._vm.devices.qemu_version >= "6.1":
```
Comment (YongxueHong) [inline provider/throttle_utils.py:144]:
```suggestion
        if self._vm.devices.qemu_version in VersionInterval("[6.1.0, )"):
```

Comment (qingwangrh) [reply provider/throttle_utils.py]: updated

**Comment by ybduan** [comment]
(1/2) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.throttle_operation_test.group_move.to_normal.q35: PASS (184.07 s)
 (2/2) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.throttle_operation_test.group_move.to_burst.q35: PASS (243.90 s)

 (1/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.throttle_operation_test.group_move.to_normal.q35: PASS (192.11 s)
 (2/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.throttle_operation_test.group_move.to_burst.q35: PASS (251.86 s)

LGTM.
Acked-by: Yanbin Duan <yduan@redhat.com>

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**